### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -115,8 +115,8 @@ commit logs available on GitHub](https://github.com/chaijs/chai/commits/master).
  * Refactor if statement out
  * Small unit test fix
  * Handle array indexing terminating paths
- * Merge pull request #309 from ericdouglas/iterableEqual-couting-once
- * couting variables just once
+ * Merge pull request #309 from ericdouglas/iterableEqual-couting -once
+ * couting  variables just once
```
Comment (astorije) [inline History.md]: Unintentional?

Comment (jsoref) [reply History.md]: I was trying to change `couting ` to `counting `, but apparently lost two characters to make that happen :-o

`cout` is a thing, but I'm prety sure that this code was talking about counting...

**Comment by jsoref** [comment]
@keithamus: dropped

**Comment by astorije** [comment]
Thanks @jsoref!

**Pull request merged**